instrumentation/opentelemetry-instrumentation-grpc/src/opentelemetry/instrumentation/grpc/__init__.py

```diff
@@ -558,14 +575,19 @@ def client_interceptor(tracer_provider=None, filter_=None):
 
     tracer = trace.get_tracer(__name__, __version__, tracer_provider)
 
-    return _client.OpenTelemetryClientInterceptor(tracer, filter_=filter_)
+    return _client.OpenTelemetryClientInterceptor(
+        tracer,
+        filter_=filter_,
+        request_hook=request_hook,
+        response_hook=response_hook,
+    )
 
 
 def server_interceptor(tracer_provider=None, filter_=None):
     """Create a gRPC server interceptor.
 
     Args:
-        tracer: The tracer to use to create server-side spans.
+        tracer_provider: The tracer to use to create server-side spans.
```

Comment:
Is this intentional?

Reply:
yes they messed up on the docstring... maybe it's not my job to fix it here now
ill revert